.github/workflows/prod.yml

```diff
@@ -1,9 +1,12 @@
 name: Publish (prod)
 
 on:
-  push:
-    tags:
-      - v*.*.*
```

Comment:
Can we add `workflow_dispatch` to the existing `push` trigger so that we can still easily publish with a version tag?

Reply:
Sure, ill re-add it